Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -829,8 +829,22 @@
 		<put-attribute name="content"
 			value="/WEB-INF/views/editing/networkshistory.jsp" />
 	</definition>
-	
-	
+
+	<definition name="base.definition.public"
```

Comment:
You are overwriting the base definition for all the public pages here. This breaks the other public pages.